### DISCUSSION TITLE
Object detection support

### DISCUSSION THREAD

**Discussion by jordond** [google/flutter-mediapipe#36] [Q&A]
Do you have any plans to add support for MediaPipe's Object Detection task?

**Comment by charlieforward9** [comment]
It looks like this is on the roadmap and prioritized below the other tasks. 

See #13 for the current progress. 

I am also waiting for this support. It would be incredible to follow the roadmap through a GitHub project if the Mediapipe dev team cared to start managing this repo as such.
